airbyte-webapp/src/pages/connections/types.ts

```diff
@@ -1,4 +1,5 @@
-export enum ConnectionSettingsRoutes {
+export const enum ConnectionPageRoutePaths {
+  ROOT = ":connectionId/*",
   STATUS = "status",
   TRANSFORMATION = "transformation",
   REPLICATION = "replication",
```

Comment:
Could we either move this to `src/pages/routePaths.tsx` or move the `DestinationPaths` defined there into a matching `src/pages/destination/types.ts` file? I have a slight preference for keeping all the route paths in one file, but I don't mind too much either way, would mostly just like consistency.